examples/trap-error/trap_error.description

```diff
@@ -0,0 +1 @@
+// The `trap` keyword will catch an error assign it to the variable.
```

Comment:
Change to "The `trap` keyword catches an error and assigns it to the variable."

Can we expand more on why it is useful etc.?